reviews/graduation-containerd.md

```diff
@@ -0,0 +1,62 @@
+**Containerd Graduation Proposal**
+
+Since the [March 2017 announcement](https://www.cncf.io/announcement/2017/03/29/containerd-joins-cloud-native-computing-foundation/) at KubeCon Berlin that containerd would be joining the CNCF, the containerd community has been busy continuing the development and testing of a stable, core container runtime for cloud native use cases.
+
+In late 2017, containerd announced its [1.0 release](https://blog.docker.com/2017/12/cncf-containerd-1-0-ga-announcement/), followed in early 2018 by a 1.1 release that has been updated with several minor fix releases through 2018. The containerd team is now preparing for the next major release, [1.2.0](https://github.com/containerd/containerd/releases/tag/v1.2.0-rc.1), with more complete Windows runtime support and a new runtime shim v2 API that is [proving](https://www.cncf.io/blog/2018/09/17/gsoc-18-kata-containers-support-for-containerd/) [valuable](https://twitter.com/resouer/status/1035066629887905792) for [runc](https://github.com/opencontainers/runc) alternatives like [Kata containers](https://katacontainers.io/).
+
+At this time, we believe containerd is ready for the [graduation stage](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#graduation-stage) within the CNCF, and per the guidelines listed there, we detail our readiness in the section below.
+
+### CNCF Graduation Criteria
+
+**Committers from at least two organizations.**
+
+Containerd has had a variety of maintainers and reviewers since its inception, and currently have 14 committers representing NTT, Huawei, Docker, Google, IBM, Microsoft, Facebook, Tesla, and Cruise Automation. We also recognize **LGTM** rights for a group we call *reviewers*, of which there are currently five representing ZTE, Docker, and independants.
```

Comment:
I am wondering if containerd has a stable governing rules for maintainers. I think governing policy for maintainers' responsibility is quite healthy for the project, since I found that some members of containerd's maintainer list have not been active for a long time.

Reply:
If I understand the comment correctly, you are right that we have some non-active maintainers. We were in the process of reaching out to them to understand their intention of continuing activity as a maintainer; the result of that is the PR that was just merged: https://github.com/containerd/project/pull/12

This does not change the make-up of the maintainer organization representation by much, but I will update the graduation PR to make it correct with the current status. Let me know if you had any other concerns here.

Reply:
Thanks for your update. Only this minor concern. Sincerely wish containerd could graduate soon.